### ISSUE TITLE
Cleanly closing UglyMol?

### ISSUE THREAD

**Issue by colinpalmer** [uglymol/uglymol#12]
Hello,

We're trying to use UglyMol for quick previews of maps and models in a new web-based GUI for CCP-EM. We're putting UglyMol into a React component so it forms just one part of a larger page. That's working nicely, except that it looks like UglyMol doesn't cleanly remove itself when we try to close it - in particular, its key bindings remain active, so subsequent keystrokes will occasionally make the whole page do something unexpected.

Does UglyMol have any support for removing itself from a page? I've looked through the code and I see that the renderer has a `dispose()` function, but I can't see anything similar for the viewer or the key bindings. If there is no support yet, would it be an easy thing to add?

I also saw the `focusable` option which looks like it should make UglyMol put its event listeners on the renderer element instead of the window. I tried that but it didn't seem to have the desired effect - but I'm not an experienced javascript programmer and maybe I was still doing something wrong! If you think that should be the right approach, I can try again and see if I can make it work.

Thanks!

**Comment by wojdyr** [comment]
I never tried removing viewer from a page.
An example that uses `focusable` is in test/multi.html
https://uglymol.github.io/test/multi.html

**Comment by colinpalmer** [comment]
Thanks, it's useful to see an example. I tried again with `focusable` set and now it seems to work properly, so I think there's actually no problem here - with `focusable = true`, UglyMol can be closed within a page and it will be cleanly removed (as far as I can tell - I haven't checked extensively for any memory leaks).

I'm not sure why it didn't work the first time I tried it - maybe I didn't refresh something properly. Anyway, thanks for your help.
